File: app/src/main/java/podtest/jiki/se/podtest/model/Broadcast.java
package podtest.jiki.se.podtest.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Broadcast {

    @SerializedName("availablestoputc")
    @Expose
    private String availablestoputc;
    @SerializedName("playlist")
    @Expose
    private Playlist playlist;
    @SerializedName("broadcastfiles")
    @Expose
    private List<Broadcastfile> broadcastfiles = null;

    public String getAvailablestoputc() {
        return availablestoputc;
    }

    public void setAvailablestoputc(String availablestoputc) {
        this.availablestoputc = availablestoputc;
    }

    public Playlist getPlaylist() {
        return playlist;
    }

    public void setPlaylist(Playlist playlist) {
        this.playlist = playlist;
    }

    public List<Broadcastfile> getBroadcastfiles() {
        return broadcastfiles;
    }

    public void setBroadcastfiles(List<Broadcastfile> broadcastfiles) {
        this.broadcastfiles = broadcastfiles;
    }

}

File: app/src/main/java/se/asapehrsson/podtest/data/Broadcast.kt
package se.asapehrsson.podtest.data

import com.google.gson.annotations.SerializedName

data class Broadcast(
        @SerializedName("availablestoputc") val availablestoputc: String? = null,
        @SerializedName("playlist") val playlist: Playlist? = null,
        @SerializedName("broadcastfiles") val broadcastfiles: List<Broadcastfile>? = null
)
